input/equateRmsWav.m

```diff
@@ -0,0 +1,79 @@
+function equateRmsWav(subjName)
+
+    if nargin < 1
+        subjName = input('Enter Subjects name: ', 's');
+    end
+
+    referenceDir = fullfile(pwd, 'Static');
+    subjectDir = fullfile(pwd, ['Motion/sub-', subjName]);
+
+    % reference folder (Static)
+    cd(referenceDir);
+    referenceWavFn = fullfile(pwd, 'Static.wav');
+
+    % Subject motion folder
+    cd(subjectDir);
+
+    targetWavFn = ['sub-', subjName, '_LRL.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+    targetWavFn = [subjName, '_RLR.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+    % Targets
+    cd(referenceDir);
+    referenceWavFn = fullfile(pwd, 'Static_T.wav');
+
+    cd(subjectDir);
+
+    targetWavFn = [subjName, '_LRL_T.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+    targetWavFn = [subjName, '_RLR_T.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+end
+
+function runFunction (referenceWavFn, targetWavFn)
```

Comment:
This looks like a good candidate to take out of this and put in CPP_PTB. 

Not urgent can be done later.